.github/workflows/code_coverage.yml

```diff
@@ -237,12 +236,11 @@ jobs:
           # Exit only if the most recent exit was a failure (by using a subshell).
           (exit $last_error_code)
 
-      - name: Run Oppia Coverage (without caching, or on a fork)
-        if: ${{ env.ENABLE_CACHING == 'false' || ((github.ref != 'refs/heads/develop' || github.event_name != 'push') && (github.event.pull_request.head.repo.full_name != 'oppia/oppia-android')) }}
+      - name: Run Oppia Coverage
         env:
           CHANGED_FILES: ${{ env.CHANGED_FILES }}
         run: |
-          bazel run //scripts:run_coverage -- $(pwd) $CHANGED_FILES --format=PROTO --processTimeout=15
+          bazel run //scripts:run_coverage -- $(pwd) $CHANGED_FILES --format=MARKDOWN --processTimeout=15
```

Comment:
Is this correct? Shouldn't it be PROTO?

Reply:
Actually, this doen't yet handle the case for proto - (it would be PROTO with the future pr (for ci) (upload comment) so when proto is declared, (with the current implementation alone) it would stop / exit after storing the protos without doing any status check, since I don't want to stop execution and creation of reports if one of the coverage reports is a failure case. I wanted the check to happen only at the very end. 

So locally -> we collect all protos, pass them to report generation and once everything is done, we do a status check and if pass : print a success statement else throw error message (while having all report generated)

With ci -> we collect all protos and return without executing the coverage reporter or status checks, - (with upcoming pr we upload and download the artifacts then finally pass the list of pb files to the coverage reporter script to generate the reports and then do a status check, so we have the report and output the status as pass or fail with providing the generated report)

Does that clarify things? or did I confuse?

Reply:
Ah I see--this seems reasonable then until the next PR is merged. Thanks for clarifying!